docs/en/pid-eaa-data-model.rst

```diff
@@ -94,15 +111,81 @@ The following claims MUST be in the JWT payload. Some of these claims can be dis
     * - **cnf**
       - [NSD].JSON object containing the proof-of-possession key materials. By including a **cnf** (confirmation) claim in a JWT, the issuer of the JWT declares that the Holder is in control of the private key related to the public one defined in the **cnf** parameter. The recipient MUST cryptographically verify that the Holder is in control of that key.
       - `[RFC7800, Section 3.1] <https://www.iana.org/go/rfc7800>`_ and Section 3.2.2.2 `SD-JWT-VC`_.
     * - **vct**
-      - [NSD].Credential type as a string, MUST be set in accordance to the type obtained from the PID/(Q)EAA Issuer metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``.
+      - [NSD].Credential type value MUST be an HTTPS URL String and MUST be set in accordance to the type obtained from the PID/(Q)EAA Issuer metadata. It serves as an identifier for the type of the SD-JWT VC and MUST be a Collision-Resistant Name as defined in Section 2 of :rfc:`7515`. It MUST contains also the number of version of the Credential type, for example, in the case of the PID, it is ``https://issuer.example.org/v1.0/personidentificationdata``.
       - Section 3.2.2.2 `SD-JWT-VC`_.
+    * - **vct#integrity**
+      - [NSD].The value MUST be an "integrity metadata" string as defined in Section 3 of [`W3C-SRI`_]. *SHA-256*, *SHA-384* and *SHA-512* MUST be supported as cryptographic hash functions. *MD5* and *SHA-1* MUST NOT be used. This claim MUST be verified according to Section 3.3.5 of [`W3C-SRI`_].
+      - Section 6.1 `SD-JWT-VC`_, [`W3C-SRI`_]
+    * - **verification**
+      - [NSD].Object containing user authentication information. It MUST contain the following sub-value:
 
+          * ``auth_trust_framework``: String identifying the trust framework used for user digital authetication.
+          * ``loa``: String identifying the level of identity assurance guarateed during the authentication process.
+          * ``auth_method``: String identifying the digital identity system used for the authentication.
+      - This specification
+
+.. note::
+
+    Credential Type Metadata JSON Document MAY be retrieved directly from URL contained in the claim **vct** using the HTTP GET method or using the vctm header parameter if provided. Unlike specified in Section 6.3.1 of `SD-JWT-VC`_ the **.well-known** endpoint is not supported in the current implementation profile.
+
+
+Digital Credential Metadata Type
+--------------------------------
+
+The Type Metadata document MUST be a JSON object and contains the following parameters.
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+      - **Reference**
+    * - **name**
+      - REQUIRED. A human-readable name of the Digital Credential type. In case of multiple language, the language tags are added to member name, delimited by a # character as defined in :rfc:`5646` (e.g. *name#it-IT*).
```

Comment:
```suggestion
      - REQUIRED. Human-readable name of the Digital Credential type. In case of multiple language, the language tags are added to member name, delimited by a # character as defined in :rfc:`5646` (e.g. *name#it-IT*).
```